### PR TITLE
Be-15-add-analitics

### PR DESCRIPTION
**Pull request by Randayy**
image

image
 
image
 
image
 
image

### CODE REVIEW

File: app/services/quiz_service.py [hunk 1]
```diff
@@ -329,18 +341,78 @@ async def get_results_of_all_users_in_company(self, company_id: UUID, current_us
...
+
+    async def get_members_and_last_quiz_submition(self, current_user: User, company_id: UUID):
+        current_user_id = current_user.id
+        await self.check_if_user_is_admin_or_owner_in_company(current_user_id, company_id)
+        all_members = await self.quiz_repository.get_all_members_of_company(company_id)
+        members_last_quiz_submition = []
+        for member in all_members:
+            member_id = member.user_id
+            last_quiz_submition = await self.quiz_repository.get_last_quiz_submition(member_id)
+            if last_quiz_submition != None:
```
Comment (Artemooon) [inline app/services/quiz_service.py:414]: if last_quiz_submition

File: app/services/quiz_service.py [hunk 2]
```diff
@@ -329,18 +341,78 @@ async def get_results_of_all_users_in_company(self, company_id: UUID, current_us
...
+
+    async def get_avarage_marks_all_members(self, current_user: User, company_id: UUID):
+        current_user_id = current_user.id
+        await self.check_if_user_is_admin_or_owner_in_company(current_user_id, company_id)
+        all_members = await self.quiz_repository.get_all_members_of_company(company_id)
+        all_members_avarage_marks = []
+        for member in all_members:
+            member_id = member.user_id
+            member_avarage_mark = await self.get_average_scores(member_id)
+            if member_avarage_mark != None:
```
Comment (Artemooon) [inline app/services/quiz_service.py:394]: if member_avarage_mark

File: app/services/quiz_service.py (hunk 1 above)
Comment (Randayy) [reply app/services/quiz_service.py:414]: fixed

File: app/services/quiz_service.py (hunk 2 above)
Comment (Randayy) [reply app/services/quiz_service.py:394]: fixed

**Pull request merged**